packages/components/pie-avatar/src/defs.ts

```diff
@@ -7,14 +7,20 @@ export interface AvatarProps {
      * Label for the username that will be turned into initials inside the avatar, if provided.
      */
     label?: string;
+
     /**
      * What HTML element the avatar should be such as button, a  or div.
      */
     tag: typeof tags[number];
```

Comment:
Just noticed this but happy for this to be addressed in a follow-up if necessary. Should this be optional? I believe we used to set all prop types as optional even if they have default values. I can't remember the exact reason though @jamieomaguire @raoufswe 
```suggestion
    tag?: typeof tags[number];
```